crypto/ml_kem/ml_kem_ipd_ref_common/indcpa.c

```diff
@@ -253,45 +259,46 @@ void indcpa_keypair_derand(uint8_t pk[KYBER_INDCPA_PUBLICKEYBYTES],
 *                                      (of length KYBER_SYMBYTES) to deterministically
 *                                      generate all randomness
 **************************************************/
-void indcpa_enc(uint8_t c[KYBER_INDCPA_BYTES],
-                const uint8_t m[KYBER_INDCPA_MSGBYTES],
-                const uint8_t pk[KYBER_INDCPA_PUBLICKEYBYTES],
-                const uint8_t coins[KYBER_SYMBYTES])
+void indcpa_enc(ml_kem_params *params,
+                uint8_t *c,
+                const uint8_t *m,
+                const uint8_t *pk,
+                const uint8_t *coins)
```

Comment:
` poly_getnoise_*` assumes `coins` is `KYBER_SYMBYTES`, so might be worth keeping the constraint.